File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        s, i = 0, 1
        while i < len(nums):
            if nums[s] == nums[i]:
                i += 1
            else:
                nums[s + 1] = nums[i]
                s += 1
                i += 1

        return s + 1

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int s = 0;
        for (int i= 1; i < nums.length; i++){
            if (nums[i] == nums[s]){
                continue;
            } 
            nums[s+1] = nums[i];
            s++;
            
        }
        return s+1;
    }
}